pinot-controller/src/test/resources/recommenderInput/KafkaPartitionRuleInput.json

```diff
@@ -72,7 +74,7 @@
         "name": "k",
         "dataType": "DOUBLE",
         "cardinality":10000,
-        "numValuesPerEntry":2,
+        "numValuesPerEntry":1,
```

Comment:
Why are we making this change in all test input files ?

Reply:
Metric columns can't be of multi-values type.